pkg/cmd/step/git/credentialhelper/step_git_credentialhelper.go

```diff
@@ -0,0 +1,159 @@
+package credentialhelper
+
+import (
+	"os"
+
+	"github.com/jenkins-x/jx/pkg/auth"
+	"github.com/jenkins-x/jx/pkg/cmd/opts/step"
+	"github.com/jenkins-x/jx/pkg/gits/credentialhelper"
+	"github.com/pkg/errors"
+
+	"github.com/jenkins-x/jx/pkg/cmd/helper"
+
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+const (
+	optionGitHubAppOwner = "github-app-owner"
+)
+
+// StepGitCredentialHelperOptions contains the command line flags
+type StepGitCredentialHelperOptions struct {
+	step.StepOptions
+
+	GitKind        string
+	GitHubAppOwner string
```

Comment:
@garethjevans, actually it might be the path is not passed in a get request. But is _GitHubAppOwner_ not effectively the organisation/owner of the repo in question? If so, the probably absolute easiest would be to pass it along. We should have all the information we need when we set the git config for the repo.

Reply:
Yeah, GitHubAppOwner is effectively the organisation/owner.  Just so I understand you correctly, you mean setting this value inside `configureJxAsGitCredentialHelper`